### PR TITLE
Home Navbar (Profile and Logout)

### PR DESCRIPTION
**Pull request by Disquietude**
Part 2 of the navbar with user profile and log out in the dropdown menu. Also, you can now submit the login and register forms by pressing Enter.

### CODE REVIEW

**Review by umbauk — APPROVED** [review]
Looks great!

File: client/src/Views/Home/Navbar/HomeNavbar.js [hunk 1]
```diff
@@ -12,26 +13,26 @@ const styles = () => ({
...
-  <Grid
-    item
-    container
-    alignItems="center"
-    justify="space-between"
-    className={props.classes.root}
-  >
-    <Grid item>
+  <Grid item container alignItems="center" className={props.classes.root}>
+    <Grid item className={props.classes.logo}>
```
Comment (skassam21) [inline client/src/Views/Home/Navbar/HomeNavbar.js:27]: we should use the hooks API for styling as opposed to the withStyles HOC - https://material-ui.com/styles/basics/#hook-api, when we are using a functional component

File: client/src/Views/Landing/Login.js [hunk 2]
```diff
@@ -47,6 +47,12 @@ class Login extends Component {
     }
   };
 
+  onEnterPress = e => {
```
Comment (skassam21) [inline client/src/Views/Landing/Login.js:50]: If you wrapped the login and signup in a form, you actually don't need to write this, and some validation will also come for free

File: client/src/Views/Home/Navbar/HomeNavbar.js (hunk 1 above)
Comment (skassam21) [reply client/src/Views/Home/Navbar/HomeNavbar.js:27]: Change this throughout this PR

Comment (Disquietude) [reply client/src/Views/Home/Navbar/HomeNavbar.js:27]: In the interest of having things ready for the presentation, I'll make a separate issue for this to work on later

**Pull request merged**